### ISSUE TITLE
Autodoc Example section on docstrings is wrongly being translated to a header in confluence while it should be a <p> tag

### ISSUE THREAD

**Issue by luizbarcelos** [sphinx-contrib/confluencebuilder#449]
Consider the `foo.py` example below:

```python
"""
This is my module

Examples
--------
>>> some module usage goes here
"""
class Bar:
    pass
```

When using the `automodule` directive, say that the module ends up as a `<h2>` in the confluence document. For some reason, the `Examples` section ends up also being a `<h2>`, even though it is part of the module. This is inconsistent with the html builder that defines the `Examples` section as a `<p>`, and ends up breaking the ToC for the PDF generated by Confluence (Examples would show up as part of the ToC - which is unexpected).

Python version: 3.7.2
Sphinx version: 3.2.1
sphinxcontrib-confluencebuilder: 1.4.0

**Comment by jdknight** [comment]
At this time, I have yet to reproduce this in a way which result in an unexpected output (unless I am overlooking something here). The sample environment built using the suggested example does show that the `Examples` section in the docstring will be created into a heading entry; however, using the `html` and `singlehtml` builders shows that they also create headings as well.

@luizbarcelos, would you mind providing any other additional hints that can help be produce the same issue you are experiencing; or correct me where I may be overlooking something?

---

### `conf.py`
```
import os
import sys

extensions = [
    'sphinx.ext.autodoc',
    'sphinx.ext.autosummary',
    'sphinxcontrib.confluencebuilder',
]

test_dir = os.path.dirname(os.path.realpath(__file__))
sys.path.insert(0, test_dir)

confluence_publish = True
...
```

### `index.rst`
```
index2
------

.. toctree::

    pagea

---

.. automodule:: foo
```

### `pagea.rst`
```
pagea
-----

.. automodule:: foo
```

### `foo.py`
```
"""
This is my module

Examples
--------
>>> some module usage goes here
"""
class Bar:
    pass
```

Confluence builder output:

> ![image](https://user-images.githubusercontent.com/1834509/114287524-7c0b6d80-9a35-11eb-9a76-8d8875c9331f.png)

HTML builder output:

> ![image](https://user-images.githubusercontent.com/1834509/114287529-8c234d00-9a35-11eb-8dd4-820f9dcc8737.png)

**Comment by jdknight** [comment]
@luizbarcelos, I finally have been able to reproduce this. It appears in some situations, autodoc processed content can sometimes build rubric nodes. Was able to reproduce this when using the `breathe` extension; where when using the above recommended example where producing section nodes. In any case, the issue is not specifically an autodocs processing issue, but a rubric node issue. Originally, if I recall, rubric nodes were processed to render in a header-like fashion (assuming to match visual styling). However, HTML-based builders seem to always produce a styled paragraph entry for these nodes. Changes introduced in #477 adjust the `confluencebuilder` extension to follow the same pattern (which is assumed to be best).

With this change, autodoc/breathe produced documentation now produces similar documentation styles between HTML builders and the builders from the `confluencebuilder` extension. This change is now available on the main repository and should be made available next stable release (v1.6). Thanks again for the report.

**Comment by jdknight** [comment]
v1.6 is now available on PyPI -- marking as closed.
